CMakeLists.txt

```diff
@@ -76,10 +76,21 @@ macro (config_hook)
     endif()
 
     # For this project
-    add_definitions(-DFMT_HEADER_ONLY)
-    include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/external/fmtlib/include)
+    find_package(fmt 7.1.3)
```

Comment:
Just for the record, the [embedded](https://github.com/OPM/opm-common/tree/aa7b619fc755ba92a4c6f49f16a780f368c9dffd/external/fmtlib/include/fmt) copy of fmtlib is [7.0.3](https://github.com/OPM/opm-common/blob/aa7b619fc755ba92a4c6f49f16a780f368c9dffd/external/fmtlib/include/fmt/core.h#L20-L21).

Reply:
thanks. was not of aware of that and just took the version in Debian.

Reply:
thanks. was not of aware of that and just took the version in Debian.